components/molecule/checkboxField/src/index.scss

```diff
@@ -1,12 +1,53 @@
 @import '~@schibstedspain/sui-theme/lib/index';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/spacing';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/color';
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/box-style';
```

Comment:
we should try to avoid importing this `/settings-compat-v7` files which where meant to give compatibility to old sui components by defining deprecated variables with a non_standard format (for instance `$m-large` instead of `$m-l`) .  Do you really need them?

Reply:
Done 😉